lib/patch-series.ts

```diff
@@ -433,6 +502,17 @@ export class PatchSeries {
         return count;
     }
 
+    protected static addSingletonHeaders(): string[] {
```

Comment:
Nit: I think I may be the one who suggested this name, but in the context where it's used as the argument to `concat()` in `const args = [...].concat(...)`, a more meaningful name would be simply `singletonHeaders()`.

Reply:
Changing to generateSingletonHeaders.